test/index.test.js

```diff
@@ -258,4 +262,71 @@ if (isWin32) {
       done();
     });
   });
+
+  describe('windows path with drive or UNC volume', function() {
+    it('should return parent dirname from absolute path with drive letter', function(done) {
+      expect(gp('C:/')).toEqual('C:/');
+      expect(gp('C:/.')).toEqual('C:/');
+      expect(gp('C:/*')).toEqual('C:/');
+      expect(gp('C:/./*')).toEqual('C:/.');
```

Comment:
Why isn't `.` removed here?

Reply:
It is because the current `glob-parent` returns `'/.'` from `'/./*'`.
I've modified to return `'/.'` from `'/./*'` by removing last `'.'` and `'./'`s in the result path.